frontend/src/components/search/SearchResultsListItem.tsx

```diff
@@ -52,87 +56,67 @@ export default function SearchResultsListItem({
-              {opportunity.opportunity_status === "archived" && (
-                <span className={metadataBorderClasses}>
-                  <strong>{t("resultsListItem.status.archived")}</strong>
-                  {opportunity?.summary?.archive_date
-                    ? formatDate(opportunity?.summary?.archive_date)
-                    : "--"}
-                </span>
-              )}
-              {(opportunity?.opportunity_status === "archived" ||
-                opportunity?.opportunity_status === "closed") &&
-                opportunity?.summary?.close_date && (
-                  <span className={metadataBorderClasses}>
-                    <strong>{t("resultsListItem.status.closed")}</strong>
-                    {opportunity?.summary?.close_date
-                      ? formatDate(opportunity?.summary?.close_date)
-                      : "--"}
-                  </span>
-                )}
-              {opportunity?.opportunity_status === "posted" && (
-                <span className={metadataBorderClasses}>
-                  <span className="usa-tag bg-accent-warm-dark">
-                    <strong>{t("resultsListItem.status.posted")}</strong>
-                    <span className="text-no-uppercase">
-                      {opportunity?.summary?.close_date
-                        ? formatDate(opportunity?.summary?.close_date)
-                        : "--"}
-                    </span>
-                  </span>
-                </span>
-              )}
-              {opportunity?.opportunity_status === "forecasted" && (
-                <span className={metadataBorderClasses}>
-                  <span className="usa-tag">
-                    <strong>{t("resultsListItem.status.forecasted")}</strong>
-                  </span>
-                </span>
-              )}
-              <span className={metadataBorderClasses}>
+            <div className="font-body-xs display-flex flex-wrap">
+              <SearchResultListItemStatus
+                archiveDate={opportunity?.summary?.archive_date}
+                archivedString={t("resultsListItem.status.archived")}
+                closedDate={opportunity?.summary?.close_date}
+                closedString={t("resultsListItem.status.closed")}
+                forecastedString={t("resultsListItem.status.forecasted")}
+                postedString={t("resultsListItem.status.posted")}
+                status={opportunity?.opportunity_status}
+              />
+              <span
+                className={`${metadataBorderClasses} tablet:order-0 order-2`}
+              >
                 <strong>{t("resultsListItem.summary.posted")}</strong>
                 {opportunity?.summary?.post_date
                   ? formatDate(opportunity?.summary?.post_date)
                   : "--"}
               </span>
+              {saved && (
+                <div className="bg-base-lighter padding-x-105 height-205 font-body-3xs">
```

Comment:
Looks like the saved indicator is a different height than the `SearchResultListItemStatus`, which uses `.usa-tag`. I wonder if this should also use the USWDS tag class so that they're the same. And if it used spans instead of divs, that might simplify the layout (the status and saved tags could both be spans in the div that controls the order).

Reply:
@andycochran `.usa-tag` doesn't affect the height, just the padding. It also sets a number of items like font color, border radius, text-transform that need to be overwritten, so I don't think it is a good candidate here:

<img width="419" alt="image" src="https://github.com/user-attachments/assets/6a1db53d-1843-46a7-a2ac-7dc66cb9203f" />